templates/_components/free_text.html.twig

```diff
@@ -1,18 +1,19 @@
-<div class="govuk-width-container">
-    <div class="govuk-border-free-text"
+    <div class="govuk-width-container">
     <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full" style="margin:100px ; margin-top:30px ; margin-right:40px ; margin-left:0px ; margin-bottom:0px"/>
     
     <div class="govuk-grid-column-one-half" >
       <div class="govuk-border-right-free-text" style="height:110px ; margin-left:20px "/>
          <div>{{ component.free_text_free_text_left_content_banner_new | raw}}</div>
          <div style="margin-top: 50px ; font-size:2px; line-height: 3px ; margin-right:8px">{{ component.free_text_free_text_left_content_new| raw}}
          </div>
-        <button class="govuk-button" data-module="govuk-button" style="margin-top:20px">Get in touch</button> 
+          {% if component.free_text_free_text_cta_label|trim is not empty and component.free_text_free_text_cta_destination|trim is not empty %}
+                 <a href="{{ component.free_text_free_text_cta_destination }}" class="govuk-button">{{ component.free_text_free_text_cta_label }}</a>
+            {% endif %}

```

Comment:
The `{% if ...` and `{% endif %}` code should start in the same column. 

E.g.

```
    {% if something %}
        <p>code</p>
    {% endif %}
```